crates/crypto/proof-params/src/lib.rs

```diff
@@ -78,8 +78,8 @@ impl Deref for LazyProvingKey {
     }
 }
 
-// Note: Conditionally load the proving key objects if the
-// bundled-proving-keys feature is enabled.
+/// Note: Conditionally load the proving key objects if the
+/// bundled-proving-keys is present.
```

Comment:
I think this slipped in during rebasing — making these doc commands will prepend them onto the doc string of the next item, which isn't what we want.

Reply:
changed